index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Youtube clone Project</title>
+  <link rel="icon" href="images/favicon.ico" type="image/x-icon">
+  <meta charset="UTF-8">
+  <meta name="description" content="YouTube Clone Project Made by Prabhakar and Luciano">
+  <meta name="keywords" content="HTML, CSS, YouTube Clone Project, The Odin Project">
+  <meta name="author" content="Prabhakar and Luciano">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link
+    href="css/style.css"
+    rel="stylesheet"
+  >
+
+</head>
+<body>
+  <header>
+    <div id="logoAndMenuDiv">
+      <img class="icon" alt="menu" src="images/ThreeLines.jpg">
+      <img class="logo" alt="YouTube Home" src="images/youtubelogo.jpg">  
+    </div>
+    <div id="searchBarDiv">
+      <input id="searchBarInput" type="text" placeholder="Your search here">
+      <button type="submit"><img class="icon" alt="Search" src="images/SearchButton.jpg"></button>
+    </div>
+    <div id="rightTopIconsDiv">
+      
+      <img class="icon" alt="Create" src="images/VideoIcon.jpg">
+      <img class="icon" alt="YouTube apps" src="images/nineDots.jpg">
+      <img class="icon" alt="Notifications" src="images/bellicon.jpg">
+      <img class="icon" alt="Account" src="images/avtar.jpg">  
+    </div>
+
+  </header>
+  <main >
+    <article class="underVideoContent">
+        <iframe id="iFrame" src="https://www.youtube.com/embed/XYeuvbhKy4I" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <h1 id="videoTitleH1">I Will Survive | Gloria Gaynor | funk cover ft. Mario Jose</h1>
+       
+        <div id=viewsLikeShare>
+          <h5>56,63,109 views * July 8, 2019</h5>
+          <img class="icon" alt="Like" src="images/LikeButton.jpg">
+          <h5>115K</h5>
+          <img class="icon" alt="Dislike" src="images/DislikeButton.jpg">
+          <h5>1.5K</h5>
+          <img class="icon" alt="Share" src="images/sharebutton.jpg">
+          <h5>SHARE</h5>
+          <img class="icon" alt="Save" src="images/saveButton.jpg">
+          <h5>SAVE</h5>  
+          <img class="icon" alt="More" src="images/threeDotsLike.jpg">
+        </div>
+        
+        <div id="channelAndSubscribeImg">
+          
+          <div class="channel">
+            <img class="channelavtar" alt="Channel" src="images/channelAvtar.jpg">
+           <div class="chnnelDetails">
+            <h4 class="channelName">Scary Pockets</h5>
+            <sub class="subcount">551K Subscribers</sub>
```

Comment:
let's give this element a padding-left to make a space between the title and the subscribers